internal/ethapi/api.go

```diff
@@ -965,7 +965,7 @@ func DoCall(ctx context.Context, b Backend, args TransactionArgs, blockNrOrHash
 	msg.TxRunMode = txRunMode
 
 	// Arbitrum: support NodeInterface.sol by swapping out the message if needed
-	if core.InterceptRPCMessage != nil {
+	{
```

Comment:
We can just pull the code out of this block now that there's no longer an if statement

Reply:
Sure, I was trying to contain the logic in a single scope but it's not necessary